components/home/WorkoutContainer.jsx

```diff
@@ -1,10 +1,20 @@
+/* eslint-disable */
```

Comment:
I know this is giving linting issues but rohith is changing this component so this might create unnecessary merge conflicts